client/checkout/woopay/express-button/express-checkout-iframe.js

```diff
@@ -269,7 +286,9 @@ export const expressCheckoutIframe = async ( api, context, emailSelector ) => {
 						return;
 					}
 					if ( response.result === 'success' ) {
-						window.location = response.url;
+						window.location = appendParamsToWooPayUrl(
```

Comment:
This only adds the parameters when the user is WooPay-authenticated and automatically redirected to WooPay.
If we end up showing the OTP model, upon a successful authentication we will receive a `redirect_to_woopay_skip_session_init`. I think we should add the params when that happens as well.

Reply:
Thank you for catching that! [122927c](https://github.com/Automattic/woocommerce-payments/pull/5903/commits/122927c9aeb4e046967f95175047fe9fb43b6522)